index.html

```diff
@@ -3294,47 +3293,6 @@ <h2>Syntaxes</h2>
 or an array of values.
       </p>
 
-      <section>
-        <h3>JSON</h3>
-
-        <p>
-The data model, as described in Section <a href="#core-data-model"></a>, can be
-encoded in JavaScript Object Notation (JSON) [[!RFC8259]] by mapping property
-values to JSON types as follows:
-        </p>
-
-        <ul>
-          <li>
-Numeric values representable as IEEE754 SHOULD be represented as a Number type.
-          </li>
-          <li>
-Boolean values SHOULD be represented as a Boolean type.
-          </li>
-          <li>
-Sequence value SHOULD be represented as an Array type.
-          </li>
-          <li>
-Unordered sets of values SHOULD be represented as an Array type.
-          </li>
-          <li>
-Sets of <a>properties</a> SHOULD be represented as an Object type.
-          </li>
-          <li>
-Empty values SHOULD be represented as a null value.
-          </li>
-          <li>
-Other values MUST be represented as a String type.
-          </li>
-        </ul>
-
-        <p class="note">
-As the transformations listed herein have potentially incompatible
```

Comment:
![felicia-when-she-left-the-village](https://user-images.githubusercontent.com/8295856/232571425-de9a5cdc-8357-4d53-8c7d-c9dba8da74c0.jpg)

Reply:
Please translate for those of us who aren't as hip to the modern meme-lingo... (Really, please don't post things like this in technical discussions. They don't tend to advance the conversation, and they are often needlessly offensive.)